src/core.js

```diff
@@ -177,6 +177,9 @@ JSONEditor.prototype = {
     });
 
     if(!classname) throw "Unknown editor for schema "+JSON.stringify(schema);
+    if(classname === "array" && JSONEditor.plugins.selectize.enable) {
+        classname = classname + "Selectize";
+    }
```

Comment:
Yep, it indeed can be done with resolvers! I've removed this now and moved that logic into a resolver of it's own